static/mmci-cli/models/NK_GK11/NK_GK11.json

```diff
@@ -107,31 +107,31 @@
     },
     {
       "name": "g",
-      "text": "g"
+      "text": "Government spending shock process"
     },
     {
       "name": "G",
-      "text": "G"
+      "text": "Government spending"
     },
     {
       "name": "Gf",
-      "text": "Gf"
+      "text": "Gf
```

Comment:
Please run `npm run test` before committing. This is not a valid json document.

fixed via 32f96c315cceddf3bcd6a9949b2dca54ea2e9673

Reply:
Oh, also a proper text editor would have caught this, you might want to have a look at something like [VS Code](https://code.visualstudio.com/)